migration/20170606-1-move-authentication-configuration-to-external-integrations.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
```

Comment:
I was a little optimistic and thought I might get more than one of these done yesterday...